packages/account/src/Components/trading-assessment/trading-assessment-new-user.tsx

```diff
@@ -30,17 +48,19 @@ const TradingAssessmentNewUser = ({
                 return { ...accumulator };
             }, {});
         }
-        onSubmit(getCurrentStep() - 1, process_form_values, null, goToNextStep, should_override);
+        onSubmit(getCurrentStep() - 1, process_form_values, goToNextStep, null, should_override);
```

Comment:
why do we change this order of params

Reply:
This is done to align with the type definition